src/lambdas/breeds-get.ts

```diff
@@ -0,0 +1,63 @@
+import fetch from 'node-fetch'
+import * as constants from './shared/constants'
+import { Response } from './shared/types'
+
+interface BreedResponse extends Response {
+  body: string[]
+}
+
+interface DogBreeds {
+  // message type is any because this can either be an object (successful response)
+  // or a string (failure)
+  message: any
```

Comment:
If it's an error, the `catch` block will get triggered, so we can type this as the happy-path payload. Even if that was not the case, we could still type this more specifically than `any`.

Reply:
I've resolved message type to be message: UnflattenedDogBreed | string